pkgs/development/tools/build-managers/waf/setup-hook.sh

```diff
@@ -18,7 +18,7 @@ wafConfigurePhase() {
 
     local flagsArray=(
         $prefixFlag
-        $wafConfigureFlags "${wafConfigureFlagsArray[@]}"
+        $configureFlags "${configureFlagsArray[@]}"
```

Comment:
here the sbindirs argument passes trought to waf, unsure about what to do to fix. :c

Reply:
cc @AndersonTorres

Reply:
It was reverted by https://github.com/NixOS/nixpkgs/pull/252874, among other things.

Reply:
oh then that revert missed the staging-next cut, would be fine to revert here as well?

Reply:
A revert against other? I didn't understand.

Reply:
I think what happened was:

1. first waf change was merged into staging
2. staging-next was cut from staging
3. second waf change reverting first waf was merged into staging-next

So the second change isn't on staging-next. :c 

So an option would be to merge this second change form staging into staging-next or reverting the first change from staging-next.

Reply:
The revert looks unnatural to me. I vote for the first.

Reply:
#254182